opts/envfile.go

```diff
@@ -31,11 +23,13 @@ func ParseEnvFile(filename string) ([]string, error) {
 		// line is not empty, and not starting with '#'
 		if len(line) > 0 && !strings.HasPrefix(line, "#") {
 			data := strings.SplitN(line, "=", 2)
-			variable := data[0]
 
-			if !EnvironmentVariableRegexp.MatchString(variable) {
-				return []string{}, ErrBadEnvVariable{fmt.Sprintf("variable '%s' is not a valid environment variable", variable)}
+			// trim the front of a variable, but nothing else
```

Comment:
Perhaps you should add some of the discussion in this PR (and the linked issue) in a comment (and in the commit message) to explain *why* we're not validating that the characters are valid for environment vars. (also to prevent someone to re-introduce the regex in future)